server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/routes/WebSocketRoutes.scala

```diff
@@ -110,10 +111,17 @@ class WebSocketRoutes @Inject() (@Named(InjectionKeys.RFC_8621) val authenticato
       .flatMap(message => handleClientMessages(context)(message))
       .doOnTerminate(context.clean)
 
-    out.sendString(
+    val applicativeMessages: SFlux[OutboundMessage] =
       SFlux.merge(Seq(responseFlux, sink.asFlux()))
-        .map(ResponseSerializer.serialize)
-        .map(Json.stringify))
+
+    val pings: SFlux[OutboundMessage] = configuration.pingInterval
+      .map(any => SFlux.interval(any)
+        .map(_ => PingMessage(???)))
```

Comment:
We can use the value 'long' generated by the ping interval to generate the ping payload